doc/msix_version.md

```diff
@@ -1,17 +1,48 @@
-## Msix Version
-
-The MSIX installer version number is used to determine updates to the app and consists of 4 numbers (`1.0.0.0`).
+## MSIX Version
 
+The MSIX installer version number is used to determine updates to the app and consists of 4 numbers (`1.0.0.0`). 
+Using the build number as the fourth number is not allowed in the windows store ([see the important banner in the microsoft documentation](https://learn.microsoft.com/en-us/windows/apps/publish/publish-your-app/package-version-numbering?pivots=store-installer-msix#version-numbering-for-windows10-packages)).
 #### The version is determined by the first available option:
 
 1. Command line `--version` flag
 2. In `pubspec.yaml`, under the `msix_config` node, the `msix_version` value
 3. Using the `version` field in `pubspec.yaml`.
    - The Pubspec version uses [semver], which is of the form `major.minor.patch-prerelease+build`
-   - `msix` will use the `major.minor.patch` and append a `0` for the MSIX version
-   - All prerelease and build info is discarded
+   - `msix` will use the `major.minor.patch.0`.
+   - All `prerelease` info is discarded
+   - NOTE: An additional flag can be enabled in the config called `version_with_build_number`. [See below for more info](#the-version_with_build_number-flag) 
 4. Fallback to `1.0.0.0`
 
 By default, if you have a valid `version` in your `pubspec.yaml` file, that will form the basis for your MSIX installer version.
 
-[semver]: https://semver.org/
\ No newline at end of file
+#### The `version_with_build_number` configuration flag:
+If true, the `build`number from the `pubspec.yaml` will be appended to `patch`: `major.minor0patch.build.0`.
```

Comment:
The 0 should only be added if patch is < 10. Think of it as 'mmmpp' - or 3 digits for the minor version and 2 digits for the patch version.

Otherwise, the limit on minor is much lower, as `1.66.10+9` would be `1.66010.10.9`

Reply:
I have updated the documentation, and i've added a second example for clarification
https://github.com/YehudaKremer/msix/blob/c2059a49b6feb5606df10147f59a68ac6ea2e95e/doc/msix_version.md?plain=1#L19-L30